default/buildings.txt

```diff
@@ -999,6 +1026,7 @@ BuildingType
             Planet environment = Adequate
         ]
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
The rationale of the changes was to disable the building when the current location condition is already unmet (Not Contains Building name = THIS_BUILDING_NAME), which is the case of BLD_TERRAFORM. Once built, the extra terraform building would hang, so I think its location condition  (and the equeuelocation) should be changed accordingly. I am not advanced enough to foresee the whole game so I agree with you about any decision about it.